src/util.jl

```diff
@@ -236,8 +236,8 @@ end
 
 @doc doc"""
 Return the index of the point x in the lexicographic order of the
-integer points of the (m-1)-dimensional simplex $\{x \mid x_0
-+ \cdots + x_{m-1} = n\}$.
+integer points of the (m-1)-dimensional simplex 
+$\{x \mid x_0 + \cdots + x_{m-1} = n\}$.
```

Comment:
Isn't it that a symbol `+` should not be placed at the beginning of a line?

Would the following work?

```
integer points of the (m-1)-dimensional simplex $\{x \mid x_0 +
\cdots + x_{m-1} = n\}$.
```

Reply:
You are correct. It worked. Thank you for pointing it out!